docs/adoption.md

```diff
@@ -94,9 +96,6 @@ Once you have added a remote backend to your Terraform and created a service pri
       | `AZURE_AD_TENANT_ID`     | This is the Directory (tenant) ID                                                                              |
       | `AZURE_SUBSCRIPTION_ID`  | Navigate to subscriptions and select the Subscription ID for your subscription                                 |
       | `STATIC_SITE_NAME`       | The name of your static site                                                                                   |
-      | `CUSTOM_DOMAIN_NAME`     | The custom domain name you want to attach to your static site.                                                 |
```

Comment:
I think we should keep `CUSTOM_DOMAIN_NAME` in the table of all GitHub Secrets that can be added. Perhaps add notation that indicates optional (or mandatory) secrets.
Also add a "see here for more details" link within the "value" column to the "Adding a custom domain" section.